src/content/news/2025-08-04-maplibre-newsletter-july-2025/index.mdx

```diff
@@ -0,0 +1,346 @@
+
+## 🗳️ Governing Board Elections 2025
+
+MapLibre’s 2025 Governing Board election [process](https://github.com/maplibre/maplibre/issues/447) has begun!
+
+- [Voting Member Nominations are now open](https://github.com/maplibre/maplibre/issues/446). Existing Voting Members can nominate new ones via pull request.
+- Governing Board Candidates can self-nominate by sharing their vision and background in the [candidate discussion thread](https://github.com/maplibre/maplibre/discussions/445).
+
+<div style="overflow-x: auto; margin-top: 1rem; margin-bottom: 1.5rem;">
+  <table style="width: 100%; border-collapse: collapse; min-width: 400px;">
+    <thead>
+      <tr>
+        <th style="text-align: left; padding: 8px 12px; font-weight: 500; color: #cfd8dc;">
+          Date
+        </th>
+        <th style="text-align: left; padding: 8px 12px; font-weight: 500; color: #cfd8dc;">
+          Milestone
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr style="border-top: 1px solid #444;">
+        <td style="padding: 10px 12px; font-weight: 600; color: #fff;">
+          Aug 15, 2025
+        </td>
+        <td style="padding: 10px 12px; color: #cfd8dc;">
+          Deadline for Voting Member nominations and Board candidate
+          registration
+        </td>
+      </tr>
+      <tr style="border-top: 1px solid #444;">
+        <td style="padding: 10px 12px; font-weight: 600; color: #fff;">
+          Aug 22, 2025
+        </td>
+        <td style="padding: 10px 12px; color: #cfd8dc;">
+          Deadline for approval of new Voting Members (via PR review)
+        </td>
+      </tr>
+      <tr style="border-top: 1px solid #444;">
+        <td style="padding: 10px 12px; font-weight: 600; color: #fff;">
+          Aug 29, 2025
+        </td>
+        <td style="padding: 10px 12px; color: #cfd8dc;">
+          Governing Board election poll closes (OpaVote)
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## 📱 MapLibre Native
+
+- **Support for Complex Scripts:** We added support for the new [`font-faces`](https://maplibre.org/maplibre-style-spec/font-faces/) property (see the [implementation PR](https://github.com/maplibre/maplibre-native/pull/3611)). This allows you to use TrueType fonts with MapLibre Native. Font files are needed for [text shaping](https://en.wikipedia.org/wiki/Text_shaping) which many complex (non-latin) scripts require. Thanks goes to Alan Wei Chen, Stefan Karschti and Grab for contributing this functionality.
+
+<div
+  style={{
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+    marginBottom: "1rem",
+  }} >
+  <div
+    style={{
+      display: "flex",
+      flexWrap: "wrap",
+      justifyContent: "center",
+      alignItems: "flex-start",
+      gap: "1rem",
+    }}
+  >
+    <Image
+      src={myanmarImg}
+      alt="Map showing Myanmar with complex script rendering"
+      style={{
+        maxWidth: "200px",
```

Comment:
Consider make them a bit bigger so you can see the text better (may just be my screen).